PyInstaller/utils/win32/winutils.py

```diff
@@ -47,6 +51,21 @@ def get_pe_file_machine_type(filename):
         return pe.FILE_HEADER.Machine
 
 
+def rewrite_exe(exe_path, data, retries=20):
+    """
+    This function writes `data` to the `exe_path` while catching OSError exceptions until `retries` are reached.
+    """
+    for retry in range(retries):
+        try:
+            with open(exe_path, 'wb') as fp:
+                fp.write(data)
+            return
+        except OSError:
+            logger.debug(f"Failed to rewrite exe, retry {retry + 1} of {retries}")
```

Comment:
Again, let's be more explicit (and use a logging level that users will actually see!).
```suggestion
            logger.info(f"IO access to {exe_path} appears to be temporarily blocked. Access attempt {retry + 1} of {retries}")
```

Reply:
In my observations the common number of required tries is 1 or 2. 
If you want to raise this to be an INFO then I'm fine with it I just didn't want to draw too much attention to a retry operation.

Reply:
Hmm, actually debug is probably fine. I want to avoid long pauses with no visible signs of activity so that people don't think it's stuck but, since even the maximum wait is short, the user not seeing anything whilst it's happening is fine.